include/boost/gil/detail/type_traits.hpp

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2017-2019 Peter Dimov.
+//
+// Distributed under the Boost Software License, Version 1.0
+// See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt
+//
+#ifndef BOOST_GIL_DETAIL_TYPE_TRAITS_HPP
+#define BOOST_GIL_DETAIL_TYPE_TRAITS_HPP
+
+#include <type_traits>
```

Comment:
You also need `boost/config.hpp` here, for `BOOST_LIBSTDCXX_VERSION`.

Reply:
Thanks. It should be fixed now.